src/main/java/uk/gov/hmcts/darts/arm/rpo/ArmRpoApi.java

```diff
@@ -35,7 +34,7 @@ boolean createExportBasedOnSearchResultsTable(String bearerToken, Integer execut
 
     List<String> getProductionOutputFiles(String bearerToken, Integer executionId, UserAccountEntity userAccount);
 
-    InputStream downloadProduction(String bearerToken, Integer executionId, String productionExportFileID, UserAccountEntity userAccount);
+    feign.Response downloadProduction(String bearerToken, Integer executionId, String productionExportFileId, UserAccountEntity userAccount);
```

Comment:
What's the rationale for returning a `feign.Response` vs an `InputStream` (from `response.body().asInputStream()`?)

Reply:
I was defering the input stream to the calling method in case there was any feign things needed besides input stream